src/app/study-flashcard/study-flashcard.component.scss

```diff
@@ -0,0 +1,18 @@
+.card-space{
+  border-style: dashed;
+  position: relative;
+  height: 620px;
```

Comment:
I learned a way you can make this fit to the screen. You can use calc(100vh - 100px). The 100px is the size of the navbar (56px) + the size of the back to view bar (44px).